Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -378,27 +408,53 @@ public final class PackageInfoMapper: PackageInfoMapping {
 
         let targets: [ProjectDescription.Target] = try packageInfo.targets
             .flatMap { target -> [ProjectDescription.Target] in
-                guard let products = targetToProducts[target.name] else { return [] }
-
-                return try platforms.compactMap { platform in
-                    try ProjectDescription.Target.from(
-                        target: target,
-                        products: products,
-                        packageName: name,
-                        packageInfo: packageInfo,
-                        packageInfos: packageInfos,
-                        packageFolder: path,
-                        packageToProject: packageToProject,
-                        productTypes: productTypes,
-                        baseSettings: baseSettings,
-                        targetSettings: targetSettings,
-                        platform: platform,
-                        minDeploymentTargets: minDeploymentTargets,
-                        targetToResolvedDependencies: targetToResolvedDependencies,
-                        targetToModuleMap: targetToModuleMap,
-                        addPlatformSuffix: platforms.count != 1
-                    )
+                
```

Comment:
```suggestion
```

Reply:
I will fix all these linting issues once I figure out what is wrong with linting setup on my machine